src/APIs/RedisAPI.ts

```diff
@@ -39,7 +40,7 @@ export const setCachedWords = async ({
   version: Version;
 }) => {
   const updatedData = assign(data);
-  updatedData.words = minimizeWords(data.words, version);
+  updatedData.words = minimizeWords(data.words as PartialWordType[], version);
```

Comment:
instead of casting `data.words` to `PartialWordType[]` on this line, we should actually change the `data: any` type on line 38 to expect its type to be `PartialWordType[]`